docs/fundamentals/jcloud/resources.md

```diff
@@ -108,6 +108,10 @@ executors:
         gpu: shared
 ```
 
+```{note}
+When using shared GPU resources, it will share the GPU memory across pods(24G memory total). If your application is memory consuming, suggest using a dedicated GPU.
```

Comment:
```suggestion
When using shared GPU resources, it will share the GPU memory across pods(24G memory total). If your application is memory consuming, we suggest using a dedicated GPU.
```